docs/content/users/install/performance.md

```diff
@@ -147,158 +156,175 @@ Instructions for Mutagen and NFS are below.
+    Mutagen provides enormous speed boosts in everyday usage, but it’s trying desperately under the hood to keep container changes reflected on the host and vice versa.
     
-    DDEV mounts a fast Docker volume onto `/var/www/html` inside the web container and then delegates to the mutagen daemon (on the host) the job of keeping all the contents of the project on the host in sync with the contents of the docker volume.
+    DDEV mounts a fast Docker volume onto `/var/www/html` inside the `web` container, then delegates to the Mutagen daemon (on the host). That daemon has the job of keeping all host project contents in sync with the contents of the Docker volume.
     
-    The strategy in the DDEV integration is to try to make sure that at key points everything is completely in sync (consistent). Consistency is a really high priority for this integration.
+    Consistency is a high priority for DDEV’s integration, which tries at key points to make sure everything is completely consistent and in sync.
     
-    The life cycle of the mutagen daemon and sync sessions are something like this:
+    The Mutagen daemon’s life cycle and sync sessions are something like this:
 
-    1. On `ddev start` the mutagen agent will be started if it's not already running.
-    2. If there is already a sync session for this project it's stopped and recreated.
-    3. On `ddev stop` and `ddev pause` the sync session is flushed (made completely consistent) and then terminated.
+    1. On `ddev start`, the Mutagen agent will be started if it’s not already running.
+    2. If there’s already a sync session for the project, it’s stopped and recreated.
+    3. On `ddev stop` and `ddev pause`, the sync session is flushed to ensure consistency, then terminated.
 
-    In addition, a synchronous flush is performed after any `ddev composer` command, because composer may cause massive changes to the filesystem inside the container, and those need to be synced before operation continues.
+    A synchronous flush happens after any `ddev composer` command, since Composer may cause massive changes to the filesystem inside the container that need to be synced before operation continues.
     
-    If you need to reset everything for a project, you can do it with `ddev mutagen reset` which starts the mutagen session from scratch and removes the docker volume so it can be recreated from scratch.
+    If you need to reset everything for a project, you can do it with `ddev mutagen reset`, which starts the Mutagen session from scratch and removes the Docker volume so it can be recreated from scratch.
 
-    ### Interaction with other usages of Mutagen
+    ### Interaction with Other Usages of Mutagen
 
     DDEV requires and provides a specific version of Mutagen, which you can see with `ddev version`.
```

Comment:
```suggestion
    DDEV requires and provides a specific version of Mutagen, which you can see with `ddev version`.  If another `mutagen` instance or daemon is installed on your workstation it doesn't matter, because DDEV's version runs separately and uses a different data directory.
```